config/database.go

```diff
@@ -90,6 +98,20 @@ func initializeConfigurationsTable(db *sqlx.DB) error {
 		return errors.Wrap(err, "failed to create ConfigurationFiles table")
 	}
 
+	// Change from TEXT (65535 limit) to MEDIUM TEXT (16777215) on MySQL. This is a
```

Comment:
I guess is already known and is not an accident, but this is executed on every single server start or command run, which mean that maybe can have side effects like quick table locks or previously build execution plans needs to be compiled again. As I said, nothing big, and actually I would keep it as is now, but pointing it to see if anybody have other point of view.

Reply:
Yeah, I couldn't come up with a clean way to do this apart from implementing a migration subsystem like the regular store. I could query the `INFORMATION_SCHEMA`, but I /feel/ like that's overkill.

I can confirm that the `MODIFY VALUE` still acquires a table lock, even if it doesn't have to change any columns. So it would briefly delay startup while a configuration update was being written. But as this isn't a high-volume table, I'm presently not too concerned and will proceed unless there are other objections.